testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -132,6 +132,13 @@ impl LocalSwarmBuilder {
             SwarmDirectory::Temporary(TempDir::new()?)
         };
 
+        // Single node orders blocks too fast which would trigger backpressure and stall for 1 sec
+        // which cause flakiness in tests.
```

Comment:
Lol, 2 fast 2 furious

Reply:
hmm, seems CI is not happy about this slowdown for whatever reason, it works pretty well locally